daemon/src/collab_settlement_maker.rs

```diff
@@ -174,6 +174,7 @@ impl Actor {
     }
 
     async fn reject(&mut self, ctx: &mut xtra::Context<Self>) {
+        // TODO: For the taker, we force-close on rejection, should we do the same thing here?
```

Comment:
I would say no. Because the maker does not have an incentive to close the position. 
If you see it from the taker's point of view: The taker wants to close because the taker clicks the close button. So, if collaborative is not an option it falls back to non-collaborative. 

The maker never clicked the close button, and should leave it up to the taker how to proceed.